code/modules/atmospherics/machinery/components/trinary_devices/filter.dm

```diff
@@ -28,9 +28,6 @@
 	if(frequency)
 		radio_connection = SSradio.add_object(src, frequency, GLOB.RADIO_ATMOSIA)
 
-/obj/machinery/atmospherics/components/trinary/filter/New()
-	..()
-
```

Comment:
Don't you just love our coders?

Reply:
this was probably me like 3 years ago tbh